aquarius/events/metadata_updater.py

```diff
@@ -268,13 +277,16 @@ def get_datatoken_pools(self, dt_address, from_block=0, to_block='latest'):
             return None
 
         pools = [get_event_data(event_abi, log).address for log in logs]
```

Comment:
could turn this into a set comprehension by replacing `[ ]` with `{ }`:
```
pools = {get_event_data(event_abi, log).address for log in logs}
```

Reply:
no, that is not going to remove duplicates.
but doing [] and then list(set([])) will do that :)

Reply:
set comprehension will remove duplicates, try it in a python console, it's the same as doing a set([]) but faster and shorter.